docker/funsize-update-generator/scripts/funsize.py

```diff
@@ -25,6 +25,9 @@
     "http://download.mozilla.org/",
 ]
 
+DEFAULT_FILENAME_TEMPLATE = "{appName}-{branch}-{version}-{platform}-" \
+                            "{locale}-{from_buildid}-{to_buildid}.partial.mar"
```

Comment:
this default template seems to differ from what would be passed by releasetasks:
{{ product }}-{{ partial_version }}-{{ appVersion }}.{locale}.{{ buildbot2ftp(platform) }}.partial.mar